openpype/hosts/nuke/api/lib_rendersettings.py

```diff
@@ -0,0 +1,92 @@
+
+from openpype.lib import Logger
+from openpype.settings import (
+    get_current_project_settings,
+    get_system_settings
+)
+
+
+class RenderFarmSettings:
+    """ Class for getting farm settings from project settings
+    """
+    log = Logger.get_logger("RenderFarmSettings")
+
+    _active_farm_module: str = None
+    _farm_modules: list = [
+        "deadline", "muster", "royalrender"]
+    _farm_plugins: dict = {
+        "deadline": "NukeSubmitDeadline"
+    }
+    _creator_farm_keys: list = [
+        "chunk_size", "priority", "concurrent_tasks"]
```

Comment:
I feel like this should maybe in the actual Deadline Module on a plug-in, see for example [here](https://github.com/ynput/OpenPype/pull/4388/files#diff-dc5f13fcc4d3823ce88f1247d130582b4feaee7afe2a018d5fe2b562ac0e937aR680-R704).

These types of settings I feel should rather be at the farm renderer module level than here. No? Isn't that what we're trying to separate out to begin with using the new publisher?

Reply:
Thanks for suggestion. It seems to be more sensible approach, thought I was looking into something more universal - more independent from a particular farm module. There is at the moment only Deadline but Royal Render will soon be fully integrated.

Nuke deadline submit plugin is not having implemented `AbstractSubmitDeadline` and it is not the part of this issue. 

Lets wait for your updates at the `AbstractSubmitDeadline` are merged and after that Nuke will follow.